### ISSUE TITLE
Prefixer falls back to prefixAll in Chromium 50

### ISSUE THREAD

**Issue by sunflowerdeath** [rofrischmann/inline-style-prefixer#84]
Prefixer falls back to prefixAll in Chromium 50:

```
new Prefixer({userAgent: 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/50.0.2661.102 Chrome/50.0.2661.102 Safari/537.36'}).prefix({display: 'flex'})
> Object {display: Array[5]}

prefixer._browserInfo
> {
	name: "Chromium",
	chromium": true,
	version: 50,
	blink: true,
	linux: true,
	x: true,
	browser: "",
	osversion: null
}

```

For example, with user agent from Firefox 46 it works:

```
new Prefixer({userAgent: 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:46.0) Gecko/20100101 Firefox/46.0'}).prefix({display: 'flex'})

> Object {display: "flex"}
```

Also, it works in Chrome.

Probably you need to add 'chromium' to this file https://github.com/rofrischmann/inline-style-prefixer/blob/master/modules/utils/getBrowserInformation.js on 4 and 10 lines.

**Comment by rofrischmann** [comment]
Do you know if Chromium and Chrome's versions always match according to features? Then we could really Chromium as an actual Chrome, else we would set version to 0 telling to use all Webkit prefixes as there's no way to keep track of all different browser versions / feature support except those listed on caniuse.com

**Comment by sunflowerdeath** [comment]
Yes, versions should always match.

> MAJOR and MINOR may get updated with any significant Google Chrome release (Beta or Stable update). MAJOR must get updated for any backwards incompatible user data change (since this data survives updates).

https://www.chromium.org/developers/version-numbers

**Comment by sunflowerdeath** [comment]
Yes, they always match.

**Comment by rofrischmann** [comment]
Fixed by @karinepires with #86
